backend/src/test/java/ca/gov/dtsstn/cdcp/api/service/UserServiceTests.java

```diff
@@ -85,4 +90,49 @@ void testCreateConfirmationCodeForUser() {
 		assertThat(confirmationCode.getExpiryDate()).isAfter(Instant.now());
 	}
 
+	@Test
+	@DisplayName("Test userService.verifyConfirmationCode(..)")
+	void testVerifyConfirmationCode_verified() {
+		final Instant creationDate = LocalDateTime.now().minusDays(73).toInstant(ZoneOffset.UTC);
+		final Instant expiryDate = LocalDateTime.now().plusDays(288).toInstant(ZoneOffset.UTC);
+		final ConfirmationCode confirmationCode = ImmutableConfirmationCode.builder().code("code value").createdDate(creationDate).expiryDate(expiryDate).build();
```

Comment:
Consider using `var` for these local variable declarations (same comment for the other tests)